src/printer.ts

```diff
@@ -523,7 +523,7 @@ function embed(
 		content = textToDoc(forceIntoExpression(textContent), {
 			...opts,
 			parser: expressionParser,
-			semi: false,
+			semi: opts.semi,
```

Comment:
Would it make sense to also change line 549 (attributes using expressions) with this change? 

(also, presumably these lines can be omitted completely due to the spread on top)

Reply:
Done!